src/styles/atoms/_atoms.button.scss

```diff
@@ -297,7 +298,8 @@
 
 .a-button,
 .a-button-negative,
-.a-button-outline {
+.a-button-outline,
+.a-button-transparent {
```

Comment:
Momenteel konden er geen icons in een `a-button-transparent` getoond worden, omdat `has-icon-left` en `has-icon-right` net geëxtend worden.